src/pantry.js

```diff
@@ -1,7 +1,81 @@
 class Pantry {
-  constructor(userIngredients) {
-    this.contents = userIngredients;
+  constructor(userData) {    
+    this.contents = typeof userData === "object" ? userData.pantry : null;
   }
+
+  returnIds(locationToSearch) {
+    return locationToSearch.map(ingredient => ingredient.id);
+  }
+
+  returnIngredient(id, ingredientList) {
```

Comment:
This is great. I think we may be use this list in some way to search by ingredients with the recipes?